src/content/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data.mdx

```diff
@@ -2256,7 +2256,7 @@ Volume data is available for volume plugins that implement the MetricsProvider i
 
 ### API server data
 
-Query the `K8sApiServerSample` event in New Relic Insights to see API Server data. For more information, see [Configure control plane monitoring](http://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring):
+Query the `K8sApiServerSample` [event](/docs/telemetry-data-platform/understand-data/new-relic-data-types/#events-new-relic) to see API Server data. For more information, see [Configure control plane monitoring](http://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/configure-control-plane-monitoring):
```

Comment:
Nitpick: Delete the http://docs.newrelic.com part of the link to `Configure control plane monitoring`